code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/PushTemplateConstants.kt

```diff
@@ -21,6 +21,8 @@ object PushTemplateConstants {
     internal const val CACHE_BASE_DIR = "pushtemplates"
     internal const val PUSH_IMAGE_CACHE = "pushimagecache"
 
+    internal const val DEFAULT_DELETE_ICON_NAME = "cross"
```

Comment:
nit: can this be moved to the `DefaultValues` object?

Reply:
Done